### ISSUE TITLE
fresh installation issue

### ISSUE THREAD

**Issue by aravindkesavarapu** [cedric05/dothttp-runner#36]
```` html
/usr/share/code/resources/app/out/vs/workbench/workbench.desktop.main.js:2084 [[object Object]]Cannot read property 'getProperties' of undefined
$onExtensionRuntimeError @ /usr/share/code/resources/app/out/vs/workbench/workbench.desktop.main.js:2084
/usr/share/code/resources/app/out/vs/workbench/workbench.desktop.main.js:2084 TypeError: Cannot read property 'getProperties' of undefined
	at t.PropertyTree.<anonymous> (/home/aravind/.vscode/extensions/shivaprasanth.dothttp-code-0.0.8/dist/extension.js:2)
	at Generator.next (<anonymous>)
	at /home/aravind/.vscode/extensions/shivaprasanth.dothttp-code-0.0.8/dist/extension.js:2
	at new Promise (<anonymous>)
	at r (/home/aravind/.vscode/extensions/shivaprasanth.dothttp-code-0.0.8/dist/extension.js:2)
	at t.PropertyTree.refresh (/home/aravind/.vscode/extensions/shivaprasanth.dothttp-code-0.0.8/dist/extension.js:2)
	at t.PropertyTree.onActiveEditorChanged (/home/aravind/.vscode/extensions/shivaprasanth.dothttp-code-0.0.8/dist/extension.js:2)
	at /home/aravind/.vscode/extensions/shivaprasanth.dothttp-code-0.0.8/dist/extension.js:2
	at g.fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:57)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:88
	at g.fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:57)
	at p.acceptDocumentsAndEditorsDelta (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:87)
	at p.$acceptDocumentsAndEditorsDelta (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:87)
	at c._doInvokeHandler (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:90)
	at c._invokeHandler (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:90)
	at c._receiveRequest (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:90)
	at c._receiveOneMessage (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:90)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:90
	at g.fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:57)
	at p.fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:65)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106
	at g.fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:57)
	at p.fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:65)
	at t._receiveMessage (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:65)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:65
	at g.fire (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:57)
	at l.acceptChunk (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:65)
	at /usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:65
	at Socket.E (/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106)
	at Socket.emit (/usr/share/code/resources/app/out/vs/code/electron-browser/workbench/events.js:315)
	at addChunk (/usr/share/code/resources/app/out/vs/code/electron-browser/workbench/_stream_readable.js:295)
	at readableAddChunk (/usr/share/code/resources/app/out/vs/code/electron-browser/workbench/_stream_readable.js:271)
	at Socket.Readable.push (/usr/share/code/resources/app/out/vs/code/electron-browser/workbench/_stream_readable.js:212)
	at Pipe.onStreamRead (/usr/share/code/resources/app/out/vs/code/electron-browser/workbench/internal/stream_base_commons.js:186)
$onExtensionRuntimeError @ /usr/share/code/resources/app/out/vs/workbench/workbench.desktop.main.js:2084
DevTools failed to load SourceMap: Could not load content for https://ticino.blob
````

![Screenshot from 2021-04-01 20-06-31](https://user-images.githubusercontent.com/61830455/113310395-d154ca00-9325-11eb-85f2-07dbd6e6878c.png)

**Comment by cedric05** [comment]
Issue seems to be happening for python3.8 installed machines.

Setting `dothttp.conf.pythonpath` to invalid path is workaround till fix is available.

**Comment by cedric05** [comment]
will be fixed via #37
